### ISSUE TITLE
Renaming Files Deletes Them

### ISSUE THREAD

**Issue by friesandacoke** [Salandora/OctoPrint-FileManager#10]
When I rename a file under the file manager tab it deletes the file. Octoprint 1.3.4 stable, Raspi 3, File Manager 0.1.2, Windows 10, Chrome Version 61.0.3163.100 (Official Build) (64-bit).

**Comment by cnlson** [comment]
yeah, i'm seeing the same thing but i think it is because i omitted .gcode and so now the file is just plain hidden.

I think there needs to be either an undo or the rename should always retain the extension. and also some sort of way to find these files would be good.
